src/Card/Card.js

```diff
@@ -0,0 +1,159 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { dcnb } from "cnbuilder";
+import { cardCtaOptions, cardAlignOptions } from "./Card.levers";
+import { FlexBox } from "../FlexBox/FlexBox";
+import { Heading } from "../Heading/Heading";
+import { CtaLink } from "../CtaLink/CtaLink";
+import { iconOptions } from "../common/icon/icon.levers";
+import { CtaButton } from "../CtaButton/CtaButton";
+import { ctaButtonVariants } from "../CtaButton/CtaButton.levers";
+import { buttonSizes } from "../common/button/button.levers";
+
+/**
+ * Card Component
+ *
+ * HTML card element
+ */
+export const Card = ({
+  className,
+  headline,
+  headingLevel,
+  superHeadline,
+  image,
+  text,
+  cta,
+  ctaText,
+  linkIcon,
+  buttonVariant,
+  buttonSize,
+  align,
+  isMinimal,
+  ...props
+}) => {
+  // Defaults & Variables.
+  // ---------------------------------------------------------------------------
+  const levers = {};
+
+  // Levers
+  // ---------------------------------------------------------------------------
+
+  let wrapperClasses =
+    "su-bg-white su-text-black su-border su-border-solid su-border-transparent-black su-shadow";
+
+  let buttonClasses = "su-rs-mt-neg1";
+
+  // Basic card image has aspect ratio 4x3 for non-round option
+  let imageWrapperClasses = "su-aspect-w-4 su-aspect-h-3";
+
+  // Option to use "minimal" card variant
+  let bodyPadding = "su-rs-px-2 su-rs-pt-2 su-rs-pb-4";
+
+  if (isMinimal) {
+    wrapperClasses = "";
+    bodyPadding = "su-rs-pt-2";
+    imageWrapperClasses = dcnb(imageWrapperClasses, { "su-rs-ml-2": false });
+  }
+
+  // Content alignment including image and CTA, default is left-align
+  // This setting overrides the alignment option in the nested CTA
+  let bodyAlign = "su-items-start";
+
+  if (align === "center") {
+    wrapperClasses = dcnb(wrapperClasses, "children:su-mx-auto su-text-center");
+    bodyAlign = "su-items-center";
+  }
+
+  return (
+    <div
+      className={dcnb("basic-card su-max-w-600 su-basefont-23", wrapperClasses)}
+    >
+      {image && (
+        <div className={imageWrapperClasses} aria-hidden="true">
+          <img src={image.filename} alt={image.alt} />
+        </div>
+      )}
+      <FlexBox
+        direction="col"
+        className={dcnb("card-body", bodyPadding, bodyAlign)}
+      >
+        {superHeadline && (
+          <span className={dcnb("su-font-serif su-bold su-mb-0")}>
```

Comment:
I imagine that it would be nice for the user to add custom classnames to each of these elements (superheadline, headline, button/link etc). Take a look at the Alert.js file and see how Shea created the "classes" object which allows for passing classnames into the individual elements 😄

Reply:
I'd actually like to talk about that pattern a bit. The classes object works but isn't super elegant. It might make more sense to break down the components like how the headless UI library does it. Headless UI doesn't just have one import to make a component but rather, an outer most component and child components that you have to nest in your own template. see how they do radiogroup https://headlessui.dev/react/radio-group

Please explore this pattern for cards as I think it is a much more flexible pattern even if it requires more up-front work from the developer to 'get all the parts' right. Storbook and the documentation should help with that problem.

Reply:
Yeah that makes sense - would be much nicer to use the nested Heading and CTAs.

Reply:
Yeah, and you could swap them out with anything much easier. In this case a lot of our variants will be different assembly of the components and we will document them as 'stories' in storyblok.